search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -65,11 +107,27 @@ function setUserSearchTopic(topic, countries) {
 }
 
 /**
- * Fetches current top trends from the backend and displays them on the website.
+ * Changes `currentTimeRange` parameter and updates trends for new time range.
+ * @param {number} timeRange The interval value for the time range.
```

Comment:
Prefer active over passive voice.

```suggestion
 * @param {number} timeRange The new time range interval value.
```